sdk/servicebus/azure-messaging-servicebus/src/main/java/module-info.java

```diff
@@ -3,6 +3,7 @@
 
 module com.azure.messaging.servicebus {
     requires transitive com.azure.core.amqp;
+    requires transitive org.reactivestreams;
```

Comment:
This should be available transitively through `com.azure.core.amqp` already.

Reply:
It is not available , thus I had to add it.

Reply:
`com.azure.core` module requires to transitively. So, anything depending on core should have it.